src/components/part/CreatePartDialog.tsx

```diff
@@ -127,6 +130,15 @@ export default function CreatePartDialog({
           size="small"
           type="text"
         />
+        <FormControlLabel
+          control={
+            <Checkbox
+              defaultChecked
+              onChange={(e) => setIndexMetadata(e.target.checked)}
+            />
+          }
+          label="Index Metadata"
+        />
       </DialogContent>
```

Comment:
We should also allow setting `suppliedInstanceIdKey`. That's how we know to carry suppliedIds through to parts.

Reply:
I agree...I only have `suppliedId` and `suppliedRevisionId` in there right now. Would need to also add `suppliedIdKey` & `suppliedRevisionIdKey`...then support the logic for selecting the different combinations.